eth/db/header.py

```diff
@@ -173,7 +177,7 @@ def _persist_checkpoint_header(
         )
         previous_score = score - header.difficulty
         cls._set_hash_scores_to_db(db, header, previous_score)
-        cls._set_as_canonical_chain_head(db, header.hash, header.parent_hash)
+        cls._set_as_canonical_chain_head(db, header, header.parent_hash)
```

Comment:
Yay that this change has so straightforward, and didn't require a new header lookup anywhere!